perun-core/src/main/java/cz/metacentrum/perun/core/entry/SecurityTeamsManagerEntry.java

```diff
@@ -63,7 +63,8 @@ public void setSecurityTeamsManagerBl(SecurityTeamsManagerBl securityTeamsManage
 	public List<SecurityTeam> getSecurityTeams(PerunSession sess) throws PrivilegeException, InternalErrorException {
 		Utils.checkPerunSession(sess);
 
-		if (AuthzResolverBlImpl.isAuthorized(sess, Role.PERUNADMIN)) {
+		if (AuthzResolverBlImpl.isAuthorized(sess, Role.PERUNADMIN) ||
+				AuthzResolver.isAuthorized(sess, Role.PERUNOBSERVER)) {
```

Comment:
You can remove "PERUNADMIN", because if is method 'isAuthorized' called by PerunAdmin (with any parameters) it should always return true. So testing for PERUNOBSERVER will also let PERUNADMIN to get data.